Source/Core/Core/PowerPC/TableGen.cpp

```diff
@@ -0,0 +1,459 @@
+#include <bitset>
+#include <cstdlib>
+#include <deque>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <string>
+#include <variant>
+#include <vector>
+
+struct Instruction
+{
+  std::string opname;
+  std::string type;
+  std::string flags;
+  std::deque<std::string> dispatch;
+  Instruction(std::string opname_, std::string type_, std::string flags_,
+              std::deque<std::string> dispatch_)
+      : opname(opname_), type(type_), flags(flags_), dispatch(dispatch_)
```

Comment:
These initializers should all `std::move` the constructor parameters into the respective struct variables.